CHANGELOG.md

```diff
@@ -1,5 +1,71 @@
 # Release Notes
 
+## Version 1.22.0-beta.0 - 2021-06-28
+
+Features:
+
+* Add auto pause on arm64 image [#11743](https://github.com/kubernetes/minikube/pull/11743)
```

Comment:
`auto-pause` addon: add support for arm64

Reply:
done